include/shisen_cpp/viewer/consumer/image_consumer.hpp

```diff
@@ -18,43 +18,36 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#ifndef SHISEN_CPP__CONSUMER__IMAGE_CONSUMER_HPP_
-#define SHISEN_CPP__CONSUMER__IMAGE_CONSUMER_HPP_
+#ifndef SHISEN_CPP__VIEWER__CONSUMER__IMAGE_CONSUMER_HPP_
+#define SHISEN_CPP__VIEWER__CONSUMER__IMAGE_CONSUMER_HPP_
 
-#include <rclcpp/rclcpp.hpp>
+#include <shisen_interfaces/msg/image.hpp>
 
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
 #include <memory>
 #include <string>
 
-#include "../node.hpp"
+#include "shisen_cpp/utility.hpp"
 
-namespace shisen_cpp
+
+namespace shisen_cpp::viewer
 {
+using Image = shisen_interfaces::msg::Image;
```

Comment:
Declare inside the class instead.

Reply:
Declare inside the class instead. Please also check the other declared method with `virtual` specifier.

Reply:
done